src/Salary.java

```diff
@@ -41,15 +42,15 @@ public double minSalary(Name[] name) {
     public double averageSum(Name[] name) {
         double sum = 0;
         double averageSum = 0;
-                for (int i = 0; i < name.length; i++) {
+        for (int i = 0; i < name.length; i++) {
             if (name[i] != null) {
                 sum += name[i].getSalary();
                 averageSum = sum / 5;
             }
         }
         return averageSum;
     }
-    }
+}
 
```

Comment:
public double averageSum(Name[] name) {
        double sum = 0;
        double counter = 0;
        for (int i = 0; i < name.length; i++) {
            if (name[i] != null) {
                sum += name[i].getSalary();
                counter++;
            }
        }
        return sum / counter;
}